FASTAI_APACHE_LICENSE

```diff
@@ -0,0 +1,60 @@
+Apache License, Version 2.0 Apache License Version 2.0, January 2004 http://www.apache.org/licenses/
```

Comment:
Apparently, we don't need this file at all.
[Here](https://www.gnu.org/licenses/license-list.html#apache2) we can read the following:

> This is a free software license, compatible with version 3 of the GNU GPL.

Furthermore, on the [official page](https://www.apache.org/licenses/GPL-compatibility.html) of Apache about this issue, we can read the following:
>The Free Software Foundation considers the Apache License, Version 2.0 to be a free software license, compatible with version 3 of the GPL. The Software Freedom Law Center provides practical advice for developers about including permissively licensed source.
Apache 2 software can therefore be included in GPLv3 projects, because the GPLv3 license accepts our software into GPLv3 works. However, GPLv3 software cannot be included in Apache projects. The licenses are incompatible in one direction only, and it is a result of ASF's licensing philosophy and the GPLv3 authors' interpretation of copyright law.

And if there is still some doubt left, we can read [here](https://www.gnu.org/licenses/gpl-faq.html#WhatDoesCompatMean) what two licenses being compatible means:

>It means that the other license and the GNU GPL are compatible; you can combine code released under the other license with code released under the GNU GPL in one larger program.

Meaning that a project under the GPLv3 LICENSE, as is ours, can have Apache 2.0 Licensed code in it.

So it should suffice having [this LICENSE](https://github.com/inspirehep/inspire-classifier/blob/master/LICENSE) in our repository (Which we already have).

P.S.: Just in case it's relevant for the future, Apache License 2.0 is compatible with project under GPLv3 License, but not the other way around. Also, it's not compatible with GPLv2, but we are using GPLv3 here.

Reply:
I still have a small confusion about whether we need to mention for any modified files if we modified them. As the APACHE license also [states](https://github.com/fastai/fastai/blob/master/LICENSE):

> You must cause any modified files to carry prominent notices stating that You changed the files

Reply:
Did you make any modifications on the code that you copied? From what I read and understand, we do have to state if any of the copied code is modified in any way, but we can do so inside the file where this code is.

Reply:
It looks fine to me then.